scripts/install-mender.sh

```diff
@@ -83,7 +108,25 @@ parse_args() {
                 fi
                 ;;
             --demo)
-                SELECTED_COMPONENTS="$DEMO_COMPONENTS"
+                SELECTED_COMPONENTS="$SELECTED_COMPONENTS $DEMO_COMPONENTS"
+                ;;
+            --commercial)
+                if ! echo "$args_copy" | grep -qe "--jwt-token"; then
+                    echo "ERROR: commercial requires --jwt-token argument."
+                    echo "Aborting."
+                    exit 1
+                fi
+                SELECTED_COMPONENTS="$SELECTED_COMPONENTS $COMMERCIAL_COMPONENTS"
+                ;;
+            --jwt-token)
+                if [ -n "$2" ]; then
+                    JWT_TOKEN=$2
```

Comment:
suggested change: JWT_TOKEN=$2 -> JWT_TOKEN="$2"

Reply:
Thanks

Reply:
Applied also to `CHANNEL` above